test/compress/arrow.js

```diff
@@ -202,3 +202,48 @@ arrow_unused_toplevel: {
     expect_stdout: [ "0", "1", "2", "9" ]
     node_version: ">=6"
 }
+
+async_identifiers: {
+    options = {
+        arrows: true,
+        ecma: 6,
+    }
+    input: {
+        let async = function(x){ console.log("async", x); };
+        let await = function(x){ console.log("await", x); };
+        async(1);
+        await(2);
+    }
+    expect: {
+        let async = x => { console.log("async", x); };
+        let await = x => { console.log("await", x); };
+        async(1);
+        await(2);
+    }
+    expect_stdout: [
+        "async 1",
+        "await 2",
+    ]
+    node_version: ">=8"
```

Comment:
Strangely enough, Node.js 0.10 fails to... fail?
https://travis-ci.org/mishoo/UglifyJS2/jobs/246339300#L242